spec/controllers/admin/videos_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+describe Admin::VideosController do
+  it {is_expected.to use_before_action(:require_user)}
```

Comment:
I have this in most of my specs. The shoulda matcher docs say:
```use_before_action``` tests that a ```before_action``` callback is defined in your controller. 

So basically just insurance that it's called. I later test for the redirect. Is it pointless to have these at all?